cypress_test/integration_tests/desktop/impress/sidebar_spec.js

```diff
@@ -43,4 +43,12 @@ describe(['tagdesktop', 'tagnextcloud', 'tagproxy'], 'Sidebar Tests', function()
 		cy.cGet('#document-container g.Page .TextParagraph .TextPosition tspan')
 			.should('have.attr', 'text-decoration', 'underline');
 	});
+
+	it('Menubutton visual test', function() {
```

Comment:
By visual test I mean picture of it, here we check the implementation what is not perfect as you will need to modify the test on every code change and visually it can be still broken.

Please see `writer/jsdialog_widgets_spec.js` -> `snapshot` function. Then you will need to put pictures generated in `integration_tests/snapshots` (just copy from `actual` folder to `base` after first run).

That way we compare screenshots of menubutton to see difference on screen, not the implementation.

Reply:
Thank you for the clarification. I have made the change